src/main/webapp/app/modules/home/filter-activity.tsx

```diff
@@ -203,6 +204,17 @@ export class FilterActivity extends React.Component<IFilterActivityProps, IFilte
     this.props.updateActivityFilter({ ...this.props.activityFilter, showPartner: !onlyShowMatching });
   };
 
+  getPartnerListValues = () => {
+    const partnerList = [];
+
+    _.forEach(this.props.selectedPartner, partnerId => {
```

Comment:
I think this.props.selectedPartner could have better naming if this contains list with partnerFilterList